src/sentry/auth/partnership_configs.py

```diff
@@ -3,6 +3,7 @@
 
 class ChannelName(Enum):
     FLY_IO = "fly"
+    FLY_DEACTIVATED = "fly-deactivated"
```

Comment:
nit: maybe name this something like `fly-disableable` to make it clear what the difference between the two providers